index.html

```diff
@@ -31,21 +31,22 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
     <link href="assets/site.css" rel="stylesheet">
     <link href="assets/globalsite.css" rel="stylesheet">
-    <link href="assets/social-share-kit.css" rel="stylesheet"/>
+    <link href="assets/social-share-kit.css" rel="stylesheet">
     <link href="assets/hlidac.css" rel="stylesheet">
     <script src="assets/vuejs-datepicker-patched.min.js"></script>
+    <script src="assets/vuejs-auto-complete-0.9.0-patched-min.js"></script>
     <script src="assets/lang-cs.js"></script>
-    <script src="assets/citycouncils.min.js"></script>
+    <script src="assets/citycouncils.min.js"></script>  <!-- seznam obecnich uradu, ale nejsou tam vsechny -->
+    <script src="assets/cityzipname.min.js"></script>   <!-- seznam psc + obci (snad vsech) pro naseptavac -->
```

Comment:
Dobrý nápad! Potom ale můžeme vyhodit `zip_name` klíč z `citycouncils(.min).js` a ušetřit trochu místa na lince.

Reply:
OK